consensus/tests/checkpoint.rs

```diff
@@ -5,8 +5,12 @@
 mod tests {
     use mina_consensus::{checkpoint::*, common::*};
     use mina_crypto::{hash::*, prelude::*};
+    use mina_rs_base::numbers::Length;
     use mina_rs_base::types::*;
     use wasm_bindgen_test::*;
+    const DEFAULT_SLOT_FILL_RATE: f64 = 0.65;
+    const DEFAULT_SLOT_FILL_RATE_DELTA: f64 = 0.15;
+    const TRANS: f64 = 4294967296_f64;
```

Comment:
where are these from?

Reply:
also a comment for each describing what they are would be good!

Reply:
See the [Mina Ocaml](https://github.com/MinaProtocol/mina/blob/af76cb7980d5e81e704120290a850ea9c6f8522e/src/lib/consensus/proof_of_stake.ml#L3948) no any document and description. Will ask O1 lab. And TRANS is define constant for 2f64.powi(32) for this [func](https://github.com/ChainSafe/mina-rs/blob/38edaa280d2c25f714481b908b9fb01209b67d2b/consensus/tests/checkpoint.rs#L72)

Reply:
Fixed.